tests/api/benchmark/test_checkout_mutations.py

```diff
@@ -228,6 +228,50 @@ def test_create_checkout(
     assert checkout_counts + 1 == Checkout.objects.count()
 
 
+@pytest.mark.django_db
+@pytest.mark.count_queries(autouse=False)
+def test_create_checkout_with_unpublished_product(
```

Comment:
Why this test is in `benchmark` folder?

Reply:
In this folder there are also other tests related to checkout like `test_create_checkout`, `test_update_checkout_lines.